go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/dtgorski/jsonlex
 
 go 1.14
+
+require github.com/brimdata/zed v1.4.0
```

Comment:
Introducing dependencies just for the sake of comparing them is not the best idea.